api/tests/Api/ContentNodes/ColumnLayout/CreateColumnLayoutTest.php

```diff
@@ -108,18 +108,11 @@ public function testCreateColumnLayoutValidatesMissingParent() {
         ]);
     }
 
-    public function testCreateColumnLayoutAllowsMissingSlot() {
```

Comment:
Should we test the opposite `testCreateColumnLayoutValidatesMissingSlot` to validate that null values are not allowed?

Reply:
Added the test for all ContentNodes in 0b4b5ed